lib/plaid/user.rb

```diff
@@ -35,13 +45,42 @@ class User
     #                             will be collected (default: today).
     #
     # Returns a Plaid::User instance.
-    def self.create(product, institution, username, password, pin = nil, options = {})
+    def self.create(product, institution, username, password, pin: nil, options: nil)
+      check_product product
+
+      payload = { username: username, password: password, type: institution.to_s }
+      payload[:pin] = pin if pin
+      payload[:options] = MultiJson.dump(options) if options
+
+      conn = Connector.new(product, auth: true)
+
+      new product, response: conn.post(payload)
     end
 
     # Public: Get User instance in case user access token is known.
     #
-    # token - the String access token for the user.
-    def self.revive(token)
+    # No requests are made, but the returned User instance is ready to be
+    # used.
+    #
+    # product - The Symbol product name you want to use, one of
+    #           Plaid::PRODUCTS (e.g. :info, :connect, etc.).
+    # token   - The String access token for the user.
+    #
+    # Returns a Plaid::User instance.
+    def self.revive(product, token)
```

Comment:
For example, when you want to save access token to a database for later use. At some point you load it back and want to make requests without calling `User.create` with all the credentials.

It's analogous to [Plaid.get_user](https://github.com/plaid/plaid-ruby/blob/master/lib/plaid.rb#L51) in the old API. But "get_user" is too common for a name.